src/randoop/JunitFileWriter.java

```diff
@@ -254,6 +270,15 @@ public static File writeDriverFile(File dir, String packageName, String driverCl
     }
     return file;
   }
+  
+ 
+  /**
+   * Creates Junit tests for the faults.
+   * Output is a set of .java files.
```

Comment:
This is how the code was built previously. All of the methods return a collection of File objects so that the names can be printed out. I removed several of these methods, so this comment may be gone.